### PR TITLE
Wrap the Tooltip

### PR DESCRIPTION
**Pull request by jordankoschei-okta**

### CODE REVIEW

File: packages/odyssey-react-mui/src/Button.tsx [hunk 1]
```diff
@@ -43,25 +45,42 @@ const Button = ({
...
+      text,
+      variant,
+    ]
   );
 
   return (
     <>
       {tooltipText && (
-        <Tooltip describeChild placement="top" title={tooltipText}>
+        <Tooltip describeChild placement="top" label={tooltipText}>
```
Comment (jordankoschei-okta) [inline packages/odyssey-react-mui/src/Button.tsx:83]: Switch `label` to `text`

File: packages/odyssey-react-mui/src/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  placement?: MuiTooltipProps["placement"];
+};
+
+export const Tooltip = ({
+  children,
+  describeChild,
+  label,
+  placement,
+}: TooltipProps) => (
+  <MuiTooltip title={label} placement={placement} describeChild={describeChild}>
```
Comment (jordankoschei-okta) [inline packages/odyssey-react-mui/src/Tooltip.tsx:32]: wtf is describeChild

Comment (KevinGhadyani-Okta) [reply packages/odyssey-react-mui/src/Tooltip.tsx:32]: My guess is `describeChild` switches between `aria-label` and `aria-description`.

We should be able to figure this out by checking for `label` or `children`. Something like that.

File: packages/odyssey-react-mui/src/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Tooltip as MuiTooltip } from "@mui/material";
+import type { TooltipProps as MuiTooltipProps } from "@mui/material";
+
+import { RefChild } from "./RefChild";
+
+export type TooltipProps = {
+  children: React.ReactElement<any, any>;
```
Comment (jordankoschei-okta) [inline packages/odyssey-react-mui/src/Tooltip.tsx:19]: Remove `React.`

Comment (jordankoschei-okta) [reply packages/odyssey-react-mui/src/Tooltip.tsx:19]: And the `any, any`

**Review by KevinGhadyani-Okta — APPROVED** [review]
Approved, but please fix the lint errors.

**Pull request merged**